src/css/lib/svg-icons/svg-icons.scss

```diff
@@ -0,0 +1,29 @@
+.svg-icon {
+	&.g-drive {
+		/* Default size for icons 16px = 1em */
+		width: 6.429em;	// Scaled based on main symbol's viewBox size: 200×1.125em÷35
```

Comment:
`rem` vs `em` :thinking:

Reply:
Not sure. I set `em` to allow to resize the icon according to its container, instead of globally, like [FontAwesome](https://github.com/FortAwesome/Font-Awesome/blob/master/css/fontawesome.css) :thinking:

Reply:
Please use [recommended rules from Josh](https://joshsanders.github.io/telicon/#get-started) as we want icons to be 16px width/height by default (at least for the telicon collection).

If you feel the need to create one sheet per collection, that's fine.